visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts

```diff
@@ -96,51 +112,78 @@ export class CodeMapLabelService implements CameraChangeSubscriber {
 		context.font = `${fontsize}px Helvetica Neue`
 
 		const margin = 20
+		const multiLineContext = message.split("\n")
 
 		// setting canvas width/height before ctx draw, else canvas is empty
 		canvas.width = context.measureText(message).width + margin
-		canvas.height = fontsize + margin
+		canvas.height = margin + fontsize * multiLineContext.length
 
 		// bg
-		context.fillStyle = "rgba(255,255,255,1)"
-		context.strokeStyle = ColorConverter.convertHexToRgba(this.storeService.getState().appSettings.mapColors.angularGreen)
+		context.font = `${fontsize}px Helvetica Neue`
+		context.fillStyle = "rgba(224,224,224,0.85)"
```

Comment:
It would be nice to have a const for the new color. Then, you could use the const in visualization/app/codeCharta/ui/codeMap/codeMap.label.service.ts too.